### DISCUSSION TITLE
solved: Dashboard misconfigured

### DISCUSSION THREAD

**Discussion by maxzapp** [netbox-community/netbox#12520] [Help Wanted!]
Hi,

after upgrading to 3.5.1 i was trying to get familiar with the new customizable dashbord. After adding in an existing DCIM widget some plugin  (topologie- view) the dashbord breaks with this message below. Unfortunatelly i dont know how to delete the pluginparts as there is not default dashborad  ore reset option .

So how can i get back to the default dash?

BR
Zappmax

==================
Server Error

There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'AttributeError'>

'Manager' object has no attribute 'restrict'

Python version: 3.8.10
NetBox version: 3.5.1
============================

**Comment by maxzapp** [comment]
More derails from my side:

if i go to https://my-ip/dashboard then i can  see the  normal navigation pane on the left side and i can access the rest of my data, there is just no dashboard anymore.

**Reply by abhi1693** [reply]
Make sure you are on the latest verison of topology views. If the issue still persists, you should open the bug with the plugin developer.

**Reply by dreng** [reply]
@abhi1693 It's not a bug with the plugin. The topology models that are shown in the dashboard are not intended to be listed or counted. Therefore they haven't got the "classic" view classes like a list view class. In fact, they shouldn't show up at all. Is it possible to add some parameter to the model in order to tell NetBox to not list it in the dashboard config?

**Comment by abhi1693** [comment]
You can reset the dashboard by going to `http://netbox/extras/dashboard/reset/`

**Comment by maxzapp** [comment]
thx to abhi1693 this worked for me:

My solution
goto https://my-ip/extras/dashboard/reset/
 there you can reset the personal dashboard
